interp/interpreter.go

```diff
@@ -17,9 +17,14 @@ func (r *runner) run(fn *function, params []value, parentMem *memoryView, indent
 	locals := make([]value, len(fn.locals))
 	r.callsExecuted++
 
+	var timeout bool
+
 	if time.Since(r.start) > time.Minute {
 		// Running for more than a minute. This should never happen.
-		return nil, mem, r.errorAt(fn.blocks[0].instructions[0], fmt.Errorf("interp: running for more than a minute, timing out (executed calls: %d)", r.callsExecuted))
+		if r.debug {
```

Comment:
For MCUs, always logging is useful, without worrying about debug flags?  If there’s a fixed flash budget to meet, would want to be able to easily track down code that interp gave up on and needs attention.